public/stylesheets/site/2.0/22-system-messages.css

```diff
@@ -37,6 +37,7 @@ div.error {
   clear: right;
     box-shadow: inset 1px 1px 2px;
     border-radius: 0.25em;
+  word-wrap: break-word;
```

Comment:
If you could nudge this in 2 more spaces so it's aligned with `border-radius` above it, that would be great! (CSS3 was new and note widely-supported when we first wrote these stylesheets, so we decided to put the new CSS3 properties at the end of rules, with an extra indent, to make it obvious where we were using them. It's a dated practice now, but consistency doesn't hurt.)

Reply:
Apologies for that, done. I debated where to align it and chose incorrectly!! Thanks for the explanation.